sdk/storage/Azure.Storage.Common/src/Shared/AesGcm/AesGcmWindows.cs

```diff
@@ -0,0 +1,143 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.Security.Cryptography;
+
+namespace Azure.Storage
```

Comment:
Should we put this all in a common namespace more like `Azure.Storage.Shared.AesGcm` or something so it's easier to lump all these pieces together via intellisense when browsing code in the future?

Reply:
I'm going to get this in one quick PR, the other comments will be in another PR.

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/29067